nixos/modules/services/mail/public-inbox.nix

```diff
@@ -109,7 +109,6 @@ let
       SystemCallArchitectures = "native";
 
       # The following options are redundant when confinement is enabled
-      RootDirectory = "/var/empty";
```

Comment:
Correct, `RootDirectory` is basically a `chroot` in the host namespace, while `TemporaryFileSystem` creates a dedicated mount namespace.